README.md

```diff
@@ -1 +1,6 @@
-# Banka
\ No newline at end of file
+# Banka
+[![Build Status](https://travis-ci.com/TamunoibiAprekuma/Banka-1.svg?branch=develop)](https://travis-ci.com/TamunoibiAprekuma/Banka-1)
+
+[![Coverage Status](https://coveralls.io/repos/github/TamunoibiAprekuma/Banka/badge.svg?branch=master)](https://coveralls.io/github/TamunoibiAprekuma/Banka?branch=master)
```

Comment:
Why is the branch set to master

Reply:
I do not really understand the question. I should make the branch develop?